action_plugins/map_to_io/__init__.py

```
@@ -267,6 +267,8 @@ def _from_xml(self, node: ElementTree.Element, library: Library) -> None:
                 node, "button-inverted", PropertyType.Bool
             )
 
+        self._io.create(self.io_input_type, self.io_input_guid)
```

What is this fixing? I looked at the logic again, this should actually raise an exception as an attempt is being made to create an input that should already exist. If that doesn't happen the entire logic before is somehow bugged.

---

Before this commit, I'd create the profile that's in the integration test, and it would work, but after saving, restarting Gremlin and reloading the profile, it wouldn't work anymore. I forget what exactly was broken, but I think the intermediate axes wasn't getting created correctly upon profile load.

---

Ahhh yeah, there is something broken with profiles loading or persisting these IO inputs. I ran into that as well when playing around with the feature a bit but didn't get to investigate. This is a case where the "failure to restore state as stored in the profile" should be fixed, as the code should really throw an exception if this is attempted twice. It's also a general failure in the IO system and we'd run into the "doesn't exist now" issue with other actions, such as Macro (once I add it in there), as well.

---

This PR no longer needs this bug to be fixed, so I have created #622 to be fixed separately.